src/Components/SmartComponents/ClusterDetail/ClusterImagesTable.cy.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { mount } from '@cypress/react18';
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import ClusterImagesTable from './ClusterImagesTable';
+import { Provider } from 'react-redux';
+import { init } from '../../../Store/ReducerRegistry';
+import images from '../../../../cypress/fixtures/clusterimageslist.json';
+import { initialState } from '../../../Store/Reducers/ClusterImagesStore';
+import {
+  CLUSTER_IMAGES_EXPORT_PREFIX,
+  CLUSTER_IMAGES_TABLE_COLUMNS,
+} from '../../../Helpers/constants';
+import {
+  itExportsDataToFile,
+  itHasNoActiveFilter,
+  itHasTableFunctionsDisabled,
+  itIsNotSorted,
+  itIsSortedBy,
+  testFilters,
+  testPagination,
+  testSorting,
+} from '../../../../cypress/utils/table';
+
+const mountComponent = () => {
+  mount(
```

Comment:
Nitpick: maybe we can refactor this to separate the cypress utility function as [here](https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/cypress/support/commands.js#L36). This would helps us to avoid code-duplication

Reply:
I looked into this -- we could, but not that well, since some tests require `MemoryRouter`, some don't. We cannot abstract `Route` since it varies greatly -- thus without major loss in code clarity we could only abstract `<Provider store={init().getStore()}>...</Provider>` which is not worth it since we would just replace one wrapper with another wrapper.

If we were to have 4 providers as Inventory it would make sense, but not here.